Calibration/HcalAlCaRecoProducers/python/alcaIsoTracksFilter_cfi.py

```diff
@@ -26,5 +25,9 @@
                                    ConeRadiusMIP     = cms.double(14.0),
                                    MinimumTrackP     = cms.double(20.0),
                                    MaximumEcalEnergy = cms.double(2.0),
-                                   IsolationEnergy   = cms.double(10.0),
+                                   MaxTrackP         = cms.double(8.0),
+                                   SlopeTrackP       = cms.double(0.05090504066),
```

Comment:
DN-2016/029 and the magic # was log(2.5)/18. It came from a study by Marina Chadeeva to define eta dependent cut